required/examples/demos/interactivity.js

```diff
@@ -27,24 +24,21 @@ var buttonPositions = [
     685, 445
 ];
 
-var noop = function () {
+function noop() {
 	console.log('click');
-};
+}
+
+for (var i = 0; i < 5; i++) {
 
-for (var i = 0; i < 5; i++)
-{
     var button = new PIXI.Sprite(textureButton);
     button.buttonMode = true;
 
     button.anchor.set(0.5);
-
-    button.position.x = buttonPositions[i*2];
-    button.position.y = buttonPositions[i*2 + 1];
+    button.x = buttonPositions[i*2];
+    button.y = buttonPositions[i*2 + 1];
 
     // make the button interactive...
     button.interactive = true;
-	
-	
 
     button
```

Comment:
Do we want to demonstrate pointer events alongside mouse and touch for interactivity guide?